docs/_includes/nav.html

```diff
@@ -36,5 +36,7 @@ <h3>Contents</h3>
             <li><a href="/iot-identity-service/api/keys-service.html">Keys Service</a></li>
             <li><a href="/iot-identity-service/api/certificates-service.html">Certificates Service</a></li>
         </ul>
+
+        <li><a href="/iot-identity-service/using-est-with-certd.html">Using EST with Certificates Service</a></li>
```

Comment:
As we're adding some more docs it might be a good time to also introduce some additional structure to our nav.  We could do this as a separate change.  My thought is that we have a "How-to guides" and an Reference section.  For example: 

* Overview
* How-to guides
  * Installing the Identity Service
  * Configuring the Identity Service
  * Managing the Identity Service
  * Protecting Keys via an HSM or TPM
    * tpm2-pkcs11
    * cryptoauthlib
    * softhsm
  * Creating an IoT agent
  * Issuing Arbitrary Certificates over EST (Advanced)
* Reference
  * Identity Service API
  * Keys Service API
  * Certificates Service API

Reply:
I updated the title, but we should change the nav in another PR.